go/service/chat_local.go

```diff
@@ -59,12 +59,98 @@ func (h *chatLocalHandler) GetThreadLocal(ctx context.Context, arg keybase1.GetT
 	return h.unboxThread(ctx, boxed, arg.ConversationID)
 }
 
+func retryUpToNTimesUntilNoError(n int, action func() error) (err error) {
```

Comment:
This is currently only used for retrying TopicIDs. Do we want to wait on that as well? Is this a rate limit concern?